platform-sdk/swirlds-common/src/main/java/com/swirlds/common/config/singleton/ConfigurationHolder.java

```diff
@@ -34,14 +33,14 @@
  * is not thread safe!
  * <p>
  * The class is defined as a singleton that provides the current configuration. By default the configuration has no
- * config sources. Therefore the default values will be used for all values of config data records (see {@link
- * ConfigData}). A new config can be set by calling the {@link #setConfiguration(Configuration)} method. That should be
- * done once in the browser or for unit tests and nowhere else!
+ * config sources. Therefore the default values will be used for all values of config data records (see
+ * {@link ConfigData}). A new config can be set by calling the {@link #setConfiguration(Configuration)} method. That
+ * should be done once in the browser or for unit tests and nowhere else!
  *
  * @deprecated Will be removed once we have the Platform Context in place or all mentioned problems have been refactored
  */
 @Deprecated(forRemoval = true)
-public final class ConfigurationHolder implements Supplier<Configuration> {
+public final class ConfigurationHolder {
```

Comment:
No idea why you did the changes in this class. From my point of view that creates no benefit but makes the PR bigger.

Reply:
I've reverted changes to this class.